### PR TITLE
RFC: Readonly directive for relationships

### PR DESCRIPTION
**Pull request by tbwiss**
Tbd

### CODE REVIEW

File: docs/rfcs/rfc-019-readonly-relationship.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+This feature request was raised in [#917](https://github.com/neo4j/graphql/issues/917).
+
+Users need a way to describe the relationship field as read-only.
+In a real-world scenario the evidence this read-only feature is missing could be: 
+```cypher
+(Post)-[createdBy:CREATED_BY]->(User)
+```
+In this case, it is reasonable to believe that `createdBy` should be immutable.
+
+A once created one-to-one (only?) relationship should, when the `@readonly` directive is present, not be able to be updated anymore.
```
[the remark this answers is not in the document]
Comment (tbwiss) [reply docs/rfcs/rfc-019-readonly-relationship.md:14]: Hmm yes this makes sense to extend it to one-to-many rels.
Just as a hint, the typeDefs above throw an error `List type relationship fields must be non-nullable and have non-nullable entries, please change type of Company.founders to [User!]!`.

File: docs/rfcs/rfc-019-readonly-relationship.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Note: A readonly relationship, in the example above `createdBy`, can still be deleted.
+
+### Usage Examples
+
+tbd
+
+## Risks
+
+What about many-to-many relationships?
```
[the remark this answers is not in the document]
Comment (tbwiss) [reply docs/rfcs/rfc-019-readonly-relationship.md:56]: I agree also with this case.

File: docs/rfcs/rfc-019-readonly-relationship.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (MacondoExpress) [reply docs/rfcs/rfc-019-readonly-relationship.md]: Thank you!

**Comment by tbwiss** [comment]
Update the rfc file number before merging this, `19` is taken..

**Comment by darrellwarde** [comment]
Closing as covered by #2867.

**Pull request closed without merging**